fineract-e2e-tests-core/src/test/resources/fineract-test-application.properties

```diff
@@ -20,6 +20,7 @@
 fineract-test.api.base-url=${BASE_URL:https://localhost:8443}
 fineract-test.api.username=${TEST_USERNAME:mifos}
 fineract-test.api.password=${TEST_PASSWORD:password}
+fineract-test.api.valid-password=${TEST_VALID_PASSWORD:A1b2c3d4e5f$}
```

Comment:
Would you mind to rename it to "strong password"? This valid password is misleading to me..

Reply:
Fixed